cmd/kubeadm/app/phases/bootstraptoken/node/token_test.go

```diff
@@ -33,7 +33,7 @@ func TestEncodeTokenSecretData(t *testing.T) {
 		{token: &kubeadmapi.TokenDiscovery{ID: "foo", Secret: "bar"}, t: time.Second}, // should use default
 	}
 	for _, rt := range tests {
-		actual := encodeTokenSecretData(rt.token.ID, rt.token.Secret, rt.t, []string{}, []string{}, "")
+		actual, _ := encodeTokenSecretData(rt.token.ID, rt.token.Secret, rt.t, []string{}, []string{}, "")
```

Comment:
We should respect the error here. Note that `actual` can be nil when `encodeTokenSecretData` validating usages fails. It will lead to panic here: `if !bytes.Equal(actual["token-id"], []byte(rt.token.ID)) {`

Reply:
@xiangpengzhao But the parameter `usages` is empty, it never will fail

Reply:
oh cool, thanks for clarifying!